src/Status/Types.elm

```diff
@@ -0,0 +1,29 @@
+module Status.Types exposing (StatusMsg(..), StatusModel, StatusResponse)
```

Comment:
I'm thinking about removing `Status` from all the types since it's kind of module name/module member stutter if someone uses `Status.Types.StatusModel`, and changing it so that when we import this module (and the others) we do:

```elm
import Status.Types as Status

foo : Status.Response -> String -> Status.Msg
foo resp str =
    -- blah blah blah
```

Thoughts?